sdk/devcenter/Azure.Developer.DevCenter/tests/DevCenterClientTestEnvironment.cs

```diff
@@ -20,7 +20,7 @@ public class DevCenterClientTestEnvironment : TestEnvironment
         private string _scope => GetRecordedVariable("DEFAULT_DEVCENTER_SCOPE");
         private string _testUserSecret => GetRecordedVariable("DEFAULT_TEST_USER_SECRET", options => options.IsSecret());
         private string _testUserName => GetRecordedVariable("DEFAULT_TEST_USER_NAME");
-        public string DevCenterName => GetRecordedVariable("DEFAULT_DEVCENTER_NAME");
+        public Uri Endpoint => new(GetRecordedVariable("DEVCENTER_ENDPOINT"));
```

Comment:
>DEVCENTER_ENDPOINT [](http://example.com/codeflow?start=56&length=18)

Curious why didn't you follow the same naming convention as the rest i.e. DEFAULT_DEVCENTER_ENDPOINT

Reply:
Not sure. Will change it back

Reply:
Switched back to the `DEFAULT_*` naming scheme!